<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VuoSubcompositionSaveAsDialog</class>
 <widget class="QDialog" name="VuoSubcompositionSaveAsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>153</red>
         <green>153</green>
         <blue>153</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>153</red>
         <green>153</green>
         <blue>153</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="96">
         <red>164</red>
         <green>166</green>
         <blue>168</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save Subcomposition to User Library</string>
   </property>
  </widget>
  <widget class="QLabel" name="nodeTitleLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Node Title</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nodeTitleLineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>140</y>
     <width>421</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QLabel" name="nodeClassNameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Lucida Grande&quot;;
color: rgb(102, 102, 102)</string>
   </property>
   <property name="text">
    <string>Node Class: nodeClassName</string>
   </property>
  </widget>
  <widget class="QLabel" name="nodeCategoryLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Category</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nodeCategoryLineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VuoSubcompositionSaveAsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VuoSubcompositionSaveAsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
